decidim-comments/config/locales/eu.yml

```diff
@@ -59,7 +65,4 @@ eu:
         loading: Iruzkinak kargatzen...
         title: "%{count} iruzkin"
       featured_comment:
-        title: Iruzkin nabarmendua
-  errors:
-    messages:
-      cannot_have_comments: ezin zaio iruzkinik egin
```

Comment:
Crowdin is having a bit of trouble with #1740. This key shouldn't be removed, I think.